website/prepare-generated-docs.mjs

```diff
@@ -29,24 +29,20 @@ if (!response.ok) {
 
 const data = await response.json();
 
-data.servers = [{
-    url: '<your-unleash-url>',
-}];
+data.servers = [
+    {
+        url: '<your-unleash-url>',
+    },
+];
 
-const outputDir = './docs/generated/'
+const outputDir = './docs/generated/';
 
 // Write the JSON to file
-const outputPath = path.join(outputDir, 'openapi.json')
+const outputPath = path.join(outputDir, 'openapi.json');
```

Comment:
This caused the linting issue from main branch hence I resolved it by rebasing and running `yarn run lint:fix` command.